app/src/main/java/ch/itomy/xendit_example/CreateTokenActivity.java

```diff
@@ -102,11 +106,11 @@ public void onError(XenditError xenditError) {
         };
 
         if (isMultipleUse) {
-            xendit.createMultipleUseToken(card, callback);
+            xendit.createMultipleUseToken(card, shouldAuthenticate, callback);
```

Comment:
We still need it when creating the token because before capture simple token, we check should_authenticate flag from the token - to allow capturing multiple use token without authentication id.